app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> None:
+    action, source_file, destination_file = command.split()
+
+    if action != "cp" or len(command.split()) != 3:
+        raise ValueError("Command in wrong format")
```

Comment:
no need to raise errors here simply rewrite that if command is wrong or file names are the same it does nothing and if everything correct it will proceed with executing rest of logic